tests/py/dynamo/converters/test_neg_aten.py

```diff
@@ -0,0 +1,24 @@
+import torch
+import torch.nn as nn
+from parameterized import parameterized
+from torch.testing._internal.common_utils import run_tests
+from torch_tensorrt.dynamo.test_utils import DispatchTestCase
+from torch_tensorrt import Input
+
+
+class TestNegConverter(DispatchTestCase):
+    def test_neg(self):
```

Comment:
Consider adding cases to check both integer and float tensor negation. It could just be two functions in the same class.

Reply:
cool will do! thanks!